src/jdk.compiler/share/classes/com/sun/tools/javac/util/RawDiagnosticFormatter.java

```diff
@@ -60,7 +60,7 @@
     /**
      * Helper class to generate stable positions for AST nodes occurring in diagnostic arguments.
      * If the AST node appears in the same line number as the main diagnostic, the line is information is omitted.
-     * Otherwise both line and column information is included, using the format @{code line:col}".
+     * Otherwise both line and column information is included, using the format {@code line:col}".
```

Comment:
Not a showstopper by any means. But while you are at it, maybe you could also fix this?

```suggestion
     * If the AST node appears in the same line number as the main diagnostic, the line information is omitted.
     * Otherwise both line and column information is included, using the format {@code line:col}.
```

Reply:
This part was recently integrated in https://github.com/openjdk/jdk/pull/6602. I assume there will be a trivial merge conflict to resolve.